sdk/js/src/User.ts

```diff
@@ -1,4 +1,5 @@
 import { DVCOptions, DVCUser, JSON } from './types'
+import 'react-native-get-random-values'
```

Comment:
will this work in non-react native?

Reply:
yup, this is just a polyfill

Reply:
oh wait.. we already have that library in the root package.json... hmmm

Reply:
we may have it but it wasnt being imported in the SDK

Reply:
will fix imports and update